packages/frames.js/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
need to export the middleware from here, see the `exports` section below. would suggest `"./middleware/neynar"`